Loop/gallery.ckcomplication/Base.lproj/ckcomplication.strings

```diff
@@ -1,16 +0,0 @@
-/* 
```

Comment:
Any change made to the complication code requires a manual regeneration of this bundle file, which can be done in the watch simulator.